.github/workflows/trunk-push.yml

```diff
@@ -49,25 +49,17 @@ jobs:
       - name: Check Out Sources
         uses: actions/checkout@v1
 
-      - name: Install JDK ${{ matrix.java }}
+      - name: Install JDK 8
         uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: 8
```

Comment:
**QUESTION:** Wouldn't it also be good to make the build matrix for later versions?